docs/using.rst

```diff
@@ -168,14 +170,19 @@ To use ADB with Host-Based tools:
 Using the box via SSH
 =====================
 
-Today most users will do their work inside the Vagrant box.  Access the box by
-using ``ssh`` to login to it with the following command::
+Today, most users will work inside the Vagrant box.  
+Access the box by using ``ssh`` to login to it with the following command::
 
     vagrant ssh
 
 You are now at a shell prompt inside the Vagrant box. You can now execute
 commands and use the tools provided.
 
+You can use the `sccli <https://github.com/projectatomic/adb-utils/blob/master/README.rst>`_
+to manage the orchestration services inside of the ADB.
+``sccli`` makes it easy to start and stop orchestration providers like Kubernetes
+or OpenShift.
+
 Using ``docker``
 ################
```

Comment:
Found some grammatical mistake under section `Using Atomic App and Nulecule` in this line
```
Note: Many Nulecule examples expect a working kubernetes environment. To setup a single node kubernetes environment use the Vagrantfile and refer the corresponding README
```.
Also, full stop is missing in the last line after `README` text.